VaticanBAV/et/BAVet196.xml

```diff
@@ -0,0 +1,308 @@
+                                </item>
+                                <item xml:id="q7" n="6">
+                                  <dim unit="leaf">8</dim>
+                                  <locus from="43r" to="50v" />
+                                </item>
+                                <item xml:id="q8" n="7">
+                                  <dim unit="leaf">4</dim>
+                                  <!-- Grebaut's assertion that this part contains 7 quaternions must be a mistake -->
+                                  <locus from="51r" to="54v" />
+                                </item>
+                              </list>
+                            </collation>
+                            <condition key="good">The lower margins show signs of being exposed to moisture.</condition>
+                          </supportDesc>
+
+                          <layoutDesc>
+                            <layout writtenLines="15" />
+                          </layoutDesc>
+                        </objectDesc>
+
+                        <decoDesc>
+                          <decoNote xml:id="d1" type="frame">
+                            <locus target="#3r"/>
+                            <desc>A decorative border with black and red lines.</desc>
+                          </decoNote>
+                        </decoDesc>
+
+                        <additions>
+                          <list>
+
+                            <item xml:id="a1">
+                              <locus target="#1v" />
+                              <desc type="Supplication">
+                                <title>ʾAsmāt Prayer</title>
+                              </desc>
+                              <q xml:lang="gez">
+                                <seg part="I">ትዮር፡ አላትዮር፡ ክትዮር፡ ፍክትዮር፡ በኃይለ፡ ዝንቱ፡ አስማቲከ፡ አንህዮ፡ ወአውርዶ፡ ለዝንቱ፡ በኖ፡ ዘሀሎ፡ ውስተ፡ ከርሱ፡ ለገብርከ፡ እገሌ፡</seg>
+                              </q>
+                            </item>
+
+                            <item xml:id="e1">
+                              <locus from="1r" to="2v" />
+                              <desc>Some scribbles in the margins in pen and pencil.</desc>
+                            </item>
+
+                          </list>
+                        </additions>
+
+                      </physDesc>
+
+                    </msPart>
+
+                    <msPart xml:id="p2">
+                      <msIdentifier><idno/></msIdentifier>
+
+                      <msContents>
+                        <summary/>
+
+                        <msItem xml:id="p2_i1">
+                          <locus from="55r" to="77v" />
+                          <title type="complete" ref="LIT2866RepCh147" />
+                          <textLang mainLang="gez" />
+                        </msItem>
+
+                        <msItem xml:id="p2_i2">
+                          <locus from="78r" to="80r" />
+                          <title>ʾAsmāt Prayer</title>
```

Comment:
it would be possible to either use the general ID LIT5888AsmatPrayer or create a new one